third_party/github.com/jhermsmeier/browserify-named-amd/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+exports_files(["LICENSE"])
```

Comment:
comment saying where it came from
see https://github.com/angular/angular/blob/master/third_party/README.md

Reply:
👍 :